docs/docs/api/event-hub.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: EventHub (WIP)
+slug: /api/EventHub
+tags:
+- module
+- public
```

Comment:
We already have event hub docs in the utils folder, we should probably migrate it here so that everything is in `docs/api/event-hub`

Reply:
I would love that. 👍 
Maybe after this PR is merged in the main? 🤔 

Maybe you have more that "just API ref doc" in this utils docs?

Reply:
Yes it's not blocking for this PR, I can take care of merging the 2 shortly after